provisioning/roles/php/tasks/main.yml

```diff
@@ -50,7 +50,7 @@
   when: ansible_os_family == "Debian"
 
 - name: Customized memory limits
-  copy: dest={{ php_mods_available }}/memory_limits.ini content="; configuration for memory limits\nmemory_limit=512M\nupload_max_filesize=1024M\npost_max_size=1024M\n"
+  copy: dest={{ php_mods_available }}/memory_limits.ini content="; configuration for memory limits\nmemory_limit = 160M\nupload_max_filesize = 1024M\npost_max_size = 1024M\n"
```

Comment:
Consider that we're talking about development mode, where all caches are turned off, symfony debug toolbar is activated (which alone consumes A LOT of memory), and all debug traces and points are activated.

I will test with your recommended settings and let you know, but up until now, in development mode, we had poor results with anything below 256M.